client/my-sites/checkout/src/components/checkout-main-content.tsx

```diff
@@ -224,9 +224,6 @@ const getPresalesChatKey = ( responseCart: ObjectWithProducts ) => {
 
 /* Include a condition for your use case here if you want to show a specific nudge in the checkout sidebar */
 function CheckoutSidebarNudge( {
-	siteId,
-	formStatus,
-	changeSelection,
```

Comment:
If you remove these props, you should remove the types (L233-235) make sure that all of the callers of `CheckoutSidebarNudge` don't pass the deprecated types.